inc/vcf/fixer.hpp

```diff
@@ -19,11 +19,7 @@
 
 #include <vector>
 #include <set>
-#include <iostream>
-#include <algorithm>
 
-#include "util/stream_utils.hpp"
-#include "util/string_utils.hpp"
```

Comment:
Why moving these includes to `fixer.cpp`?

Reply:
Maybe i moved them because they weren't needed in that hpp file... but are needed in `fixer.cpp`
Should i revert it?

Reply:
the current style is keeping most of the includes in hpp files if possible. Exceptions are done in cases of cyclic dependencies and similar.

Nonetheless, moving some includes to cpp files could reduce recompilation times (example: if some a.hpp changed, but it is only used in b.cpp, and c.cpp includes b.hpp, c.cpp will be recompiled without need), although I tinkered with this in the past and IMO **no changes should be made to optimize compilation times without actual measuring** as it is not obvious how the compilations times will change.